draft-irtf-cfrg-hash-to-curve.md

```diff
@@ -1300,7 +1338,7 @@ Steps:
 19.   y = CMOV(t4, t3, e3)   // if e2 == True, y = t3, else y = t4
 20.  e4 = sgn0(u) == sgn0(y)
 21.   y = CMOV(-y, y, e4)
-22. Output clear_h(x, y)
+22. Output (x, y)
```

Comment:
nit: "Output" -> "return" for consistency?

Reply:
good catch